_rules/important-line-height-wide-enough-78fd32.md

```diff
@@ -0,0 +1,371 @@
+---
+id: 78fd32
+name: Important line height in style attributes is wide enough
+rule_type: atomic
+description: |
+  This rule checks that the `style` attribute is not used to prevent adjusting `line-height` by using `!important`, except if it's at least 1.5 times the font size.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  previous_authors:
+    - Jey Nandakumar
+  funding:
+    - WAI-Tools
+---
+
+## Applicability
+
+This rule applies to any [HTML element][] with one or more [visible][] [text node][] child, when the text inside the elements includes a [soft wrap break][] and all the following are true for the `line-height` property of the element:
```

Comment:
```suggestion
This rule applies to any [HTML element][] with one or more [visible][] [text node][] children, when the text inside the elements includes a [soft wrap break][] and all the following are true for the `line-height` property of the element:
```
Could also do `with one or more [visible][] child [text nodes][]`, but I think a plural needs to be in there somewhere. Goes for the important word spacing rule as well.